apps/client/src/app/pages/transactions/transactions-page.component.ts

```diff
@@ -175,19 +179,19 @@ export class TransactionsPageComponent implements OnInit {
     this.unsubscribeSubject.complete();
   }
 
-  private openCreateTransactionDialog(): void {
+  private openCreateTransactionDialog(aTransaction?: OrderModel): void {
     const dialogRef = this.dialog.open(CreateOrUpdateTransactionDialog, {
       data: {
         accounts: this.user?.accounts,
         transaction: {
           accountId: this.user?.accounts.find((account) => {
```

Comment:
I think this would also make sense to duplicate:

```
accountId: aTransaction?.accountId ?? this.user?.accounts.find((account) => {
  return account.isDefault;
})?.id,
```

Reply:
Yep! Could make sense also the type? @dtslvr

Reply:
Yes I think so 😃

Reply:
Hmm, you need `dataSource` as well!

```
dataSource: aTransaction?.dataSource ?? null,
```

It's a hidden field and normally set via the autocomplete...

Reply:
Done! 👍